mono/metadata/file-mmap.h

```diff
@@ -18,18 +18,25 @@
 #include <mono/metadata/object-internals.h>
 #include <mono/utils/mono-compiler.h>
 
-extern void mono_mmap_close (void *mmap_handle);
+void
```

Comment:
I don't know if it's alright to remove `extern` from these icall declarations.

Reply:
Extern is pretty meaningless, noise, just messes up the code.